### ISSUE TITLE
checkpoint_exclude_scopes?

### ISSUE THREAD

**Issue by lily10086** [DeepSegment/ConvNet-Segment#6]
Hello,
       I don't understand why checkpoint_exclude_scopes is  'InceptionV3/Logits,InceptionV3/AuxLogits',
       Whether I can or not  set it to None?
       And what's the use of  'ignore_missing_vars'?
Thanks.

**Comment by txzhao** [comment]
Hi,

'checkpoint_exclude_scopes' denotes those parameters we do not need during training. If you notice in the original literature or the [pipeline](https://github.com/DeepSegment/FCN-GoogLeNet#pipeline) part in our readme file, the whole network should first be convolutionalized, which means that there is only one type of layer, namely convolutional layers, in network. This is why we want to discard those parameters from 'InceptionV3/Logits,InceptionV3/AuxLogits'.

For more details, you can find them in the two links below:
1. https://datascience.stackexchange.com/questions/17522/fine-tuning-a-model-from-an-existing-checkpoint-with-tensorflow-slim
2. https://www.tensorflow.org/tutorials/image_retraining

**Comment by lily10086** [comment]
Thanks,You can always give me much help.

**Comment by txzhao** [comment]
No problem. Glad to help!
